contracts/drafts/Strings.sol

```diff
@@ -0,0 +1,52 @@
+pragma solidity ^0.5.0;
+
+/**
+ * @title Strings
+ * @dev String operations.
+ * via https://github.com/oraclize/ethereum-api/blob/master/oraclizeAPI_0.5.sol MIT licence
+ */
+library Strings {
```

Comment:
A library is the right call, but we should make the functions `internal`, so that they are inlined in the calling contract, as opposed to `delegatecall`ed into from an external contract.

Reply:
Note that you'll need to write a mock contract to test it. See e.g. how `Math.sol` is tested, using `MathMock.sol`.